kratos/CMakeLists.txt

```diff
@@ -57,7 +57,7 @@ set( KRATOS_CORE_SOURCES
     ${CMAKE_CURRENT_SOURCE_DIR}/processes/calculate_nodal_area_process.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/processes/skin_detection_process.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/processes/replace_elements_and_condition_process.cpp
-    ${CMAKE_CURRENT_SOURCE_DIR}/processes/vtk_output_process.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/processes/vtk_output.cpp
```

Comment:
Why changing name?, if it is a process should have process in the name (my opinion)

Reply:
because it is not a process anymore. it is derived from IO. By the way where should this go then, into `input_output` folder ?

Reply:
Yep

Reply:
this is what was suggested right ?

Reply:
Yes, Ok, so it only remains to change the folder

Reply:
yes. I am confused to put it in the `input_output` folder because there is no IO there ! all there are some loggers !

Reply:
I would still put it there
Just because the other ones are not there yet doesn't mean they are correct :)

Reply:
It is because legacy reasons. Moving all the files there will require extra work, I assume will be moved eventually

Reply:
Done !